.github/actions/get-solc/action.yml

```diff
@@ -34,4 +34,4 @@ runs:
       run: |
         echo "$(pwd)/solc" >> $GITHUB_PATH
         chmod +x solc/solc
-        xattr -x solc/solc
```

Comment:
this will force you to get a +1 from release team I believe, I would remove this change